packages/transaction-controller/src/types.ts

```diff
@@ -883,7 +883,7 @@ export interface RemoteTransactionSourceRequest {
   /**
    * The IDs of the chains to query.
    */
-  chainIds: Hex[];
+  chainIds?: Hex[];
```

Comment:
Can we remove this completely since it's no longer used by our only remote transaction source?

Reply:
fixed